apps/teams-test-app/e2e-test-data/pages.json

```diff
@@ -22,9 +22,25 @@
       "type": "callResponse",
       "boxSelector": "#box_returnFocus",
       "checkboxState": true,
-      "expectedAlertValue": "returnFocus called with true",
+      "version": ">2.27.0",
```

Comment:
This `version` is Teams-Js library version, I would suggest to change it from '>2.27.0' to  '>= 2.27.0' or you could say '>2.26.0'